.github/workflows/ci.yml

```diff
@@ -59,6 +49,33 @@ jobs:
       with:
         profile: minimal
         toolchain: 1.42.0
-        components: clippy
     - name: Run tests
       run: cargo test --verbose --features base64_format,trace,metrics,serialize,binary_propagator,tokio,async-std,serde,http,tonic,reqwest -p opentelemetry -p opentelemetry-jaeger -p opentelemetry-zipkin
+  meta:
+    continue-on-error: true
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+      with:
+        submodules: true
+    - uses: actions-rs/toolchain@v1
+      with:
+        toolchain: nightly
+        components: rustfmt, clippy
+        override: true
+    - run: cargo fmt -- --check
+    - uses: arduino/setup-protoc@v1
+    - uses: actions-rs/clippy-check@v1
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+        args: --all-features --all
+    - uses: actions-rs/cargo@v1
+      with:
+        command: test
+        args: --all-features --no-fail-fast -p opentelemetry -p opentelemetry-semantic-conventions -p opentelemetry-jaeger
+      env:
+        CARGO_INCREMENTAL: '0'
+        RUSTFLAGS: '-Zprofile -Ccodegen-units=1 -Cinline-threshold=0 -Clink-dead-code -Coverflow-checks=off -Cpanic=abort -Zpanic_abort_tests'
+        RUSTDOCFLAGS: '-Zprofile -Ccodegen-units=1 -Cinline-threshold=0 -Clink-dead-code -Coverflow-checks=off -Cpanic=abort -Zpanic_abort_tests'
+    - uses: actions-rs/grcov@v0.1
```

Comment:
We could use the latest version of grcov here - v0.1.5

Reply:
Looks like this is the latest? (v0.1 is pointing at same as v0.1.6) https://github.com/actions-rs/grcov/tags